src/Nav/index.js

```diff
@@ -12,32 +12,45 @@ governing permissions and limitations under the License.
 import React from 'react'
 import PropTypes from 'prop-types'
 import { withPrefix } from 'gatsby'
-import { defaultFocus } from '../ManifestUtils'
 import { isExternal } from '../utils'
 
 import '@spectrum-css/sidenav'
 import './index.css'
 
-const nav = (data, defaultFocus, currDepth, maxDepth) => {
+const isPathSelected = (path, selected) => {
+  const compare = !selected.endsWith('/')
+    ? selected
+    : selected.substring(0, selected.length - 1)
+  return path === compare
+}
+
+const nav = (data, selected, currDepth, maxDepth) => {
   if (currDepth >= maxDepth) {
     return ''
   }
 
   return (
     <ul className='spectrum-SideNav  spectrum-SideNav--multiLevel'>
```

Comment:
There is a redundant space after `SideNav`

Reply:
I think that for these kinds of things we should create issues for them, and then address them as linting rules for the future if possible. For a few reasons:

1. to keep the purpose of PR concise and prevent bloating (in this particular case, the `className` format is code that already existed and does not need to be changed, even if it aesthetically would be better to change it)
2. to keep an automation-first mindset when it comes to menial changes/nits like this (i.e.: lint it, don't call it out over and over)
3. building on 🔼 - shifting towards a perspective of putting the _purpose_ of the PR first is more productive and less mentally taxing for the engineer opening the PR. Or everyone involved, IMO

But I'll defer to @macdonst & @hirenoble here since it's a larger discussion point.

Reply:
I'll fix that.